addon/components/column-chart/component.js

```diff
@@ -95,13 +99,25 @@ export default Ember.Component.extend({
 
         const xAxis = this.get('xAxis');
         const yAxis = this.get('yAxis');
+        const xTimeScale = d3.time.scale().domain(xAxis.domain);
+
+        const indicatorDate = this.get('currentInterval') ? this.get('currentInterval.start._d') : null;
+        const showCurrentIndicator = this.get('showCurrentIndicator');
 
         const titles = this.get('series').map(entry => entry.title);
 
         const data = this.get('data');
         const tooltipDateFormat = this.get('tooltipDateFormat');
         const formatter = this.get('xAxis.formatter') || (value => value);
 
+        function isIntervalIncluded(ticks, interval) {
+            return ticks.toString().includes(interval.toString());
```

Comment:
Unfortunately `includes` is not supported in IE 11

Reply:
Should I substitute this for a for loop, then? Or is there a more efficient way to do it that IE 11 supports?

Reply:
I would use something like `ticks.toString().indexOf(interval.toString()) > -1`

Reply:
Makes sense. I always hated that structure because it feels hack-y, but IE 11 makes sure nothing is that easy 😞  I'll push some changes up here in a bit